src/displayapp/screens/ApplicationList.h

```diff
@@ -21,14 +23,31 @@ namespace Pinetime {
         bool OnTouchEvent(TouchEvents event) override;
 
       private:
+        auto CreateScreenList() const;
+        std::unique_ptr<Screen> CreateScreen(unsigned int screenNum) const;
+
         Controllers::Settings& settingsController;
         Pinetime::Controllers::Battery& batteryController;
         Controllers::DateTime& dateTimeController;
 
-        ScreenList<2> screens;
-        std::unique_ptr<Screen> CreateScreen1();
-        std::unique_ptr<Screen> CreateScreen2();
-        // std::unique_ptr<Screen> CreateScreen3();
+        static constexpr int appsPerScreen = 6;
+        static constexpr std::array<Tile::Applications, appsPerScreen * 2> applications {{
```

Comment:
`nScreens` is explicitly mentioned here as 2. If I want to add another app to the list I need to add the `2` to `3`, which is equivalent to `nScreens`.

We could explicilty set `nScreens` and then calculate the `applications.size()` instead

If I just add an `+1` to add the new application as the only app on the third screen then I get an error at runtime when I swipe to the third screen